integration/requirements.txt

```diff
@@ -1,15 +1,24 @@
+# NOTE: make sure to update the requirements-frozen.txt file when making a change here
+# Sort this file lexicographically
+# Generate the requirements-frozen.txt file as follows:
+#  pip install -r requirements.txt
+#  pip freeze > requirements-frozen.txt
```

Comment:
Does this need to be in a "clean" environment, like a fresh virtualenv where I haven't already pip installed other stuff not needed for the integration tests?

Reply:
it doesn't really matter. if you get a green run it's good.

Reply:
The key thing is that every dependency is frozen

Reply:
If I have pip installed a bunch of extra garbage that the tests don't need, they will get included in this file, right? That seems wrong.

Reply:
I don't think it matters much if the tests run, but I'll say to do it from Vagrant

Reply:
updated the instructions. you were right - I was able to pull things in that at least Travis couldn't install. that's a problem